src/fields/field_interpolated_p0.cc

```diff
@@ -70,7 +70,7 @@ void FieldInterpolatedP0<spacedim, Value>::init_from_input(const Input::Record &
 
 	// read mesh, create tree
     {
-       source_mesh_ = new Mesh();
+       source_mesh_ = new Mesh(rec);
```

Comment:
- can not use rec since it is for the main mesh file.
- modify this Mesh constructor to accept also empty record accessor and in that case call ReaderToStorage for `"{mesh_file=\"\"}"`.
- if empty accessor is not allowed, just call this here